person.rb

```diff
@@ -26,4 +26,8 @@ def of_age?
   def correct_name
     @name
   end
+
+  def add_rental(date, book)
+    @rentals.push(Rental.new(date, self, book)) unless @rentals.includes?(Rental.new(date, self, book))
+  end
```

Comment:
- [ ] you have to just create the rental it will automatically add it to rental when it call `initialize` method. You can do just what I have suggested in `Book` class.